libraries/chain/util/smt_token.cpp

```diff
@@ -79,16 +79,12 @@ share_type payout( database& db, const asset_symbol_type& symbol, const account_
    for ( auto& _asset : assets )
    {
       if ( _asset.symbol.is_vesting() )
-      {
```

Comment:
Is there a reason to remove these brackets? This specific remove make this code less readable and subtly changes the behavior here. It is not immediately clear to me why this should be changed.

Reply:
According to `validate_database_invariants` I needed to add vesting to the total supply on the token for it to add up.

Reply:
Got it, I understand now. This was probably written in an earlier vesting of the SMT scope prior to the SMT vesting destination.

Curious why this wasn't causing a failure earlier... Even awarding Vesting SMTs with a 1:1 ratio should have caused problems...